apps/events-helsinki/src/domain/search/eventSearch/constants.tsx

```diff
@@ -96,28 +96,28 @@ export enum EVENT_SEARCH_FILTERS {
 export const CATEGORY_CATALOG = {
   [EventTypeId.Course]: {
     default: [
```

Comment:
Could this list be shortened just: `default: Object.values(EVENT_CATEGORIES)` ?

Reply:
At the moment EVENT_CATEGORIES.MISC is missing from the list here (That's how it was in the events-helsinki-ui repository also). EVENT_CATEGORIES.MISC and MISC_KEYWORDS could probably be removed altogether though as unused.

Reply:
Taken into account by the force pushed version